### PR TITLE
restyle, rephrase, more explanation to the congiguration file added, …

### LINKED ISSUE #2250: Installation guide - Elasticsearch cluster improvements
The goal is to test and verify the **Elasticsearch cluster installation** section of a New Documentation Structure.

### PR DESCRIPTION
**Pull request by dkempny**
Hello Team,
This PR closes #2250 . 

The modified files include an improved explanation of Elasticsearch configuration, rephrasing, restyling and references to the Elasticsearch documentation to help the user to better uderstand the process.

All the best,
Daria

### CODE REVIEW

File: source/_templates/installations/elastic/common/elastic-multi-node/configure_elasticsearch.rst [hunk 1]
```diff
@@ -20,14 +20,16 @@ The file ``/etc/elasticsearch/elasticsearch.yml`` has to be edited:
           - <elasticsearch_ip_node2>
           - <elasticsearch_ip_node3>
 
+Depending on the node type, some parameters may vary between nodes. ``cluster.initial_master_nodes`` and ``discovery.seed_hosts`` are lists of all the master-eligible nodes in the cluster. The parameter ``node.master: false`` must be included in every Elasticsearch node that will not be configured as master.
+
 Values to be replaced:
 
-- ``<elasticsearch_ip>``: the host IP. I.e: ``10.0.0.2``. The value ``0.0.0.0`` is an acceptable IP address and will bind to all network interfaces.
+- ``<elasticsearch_ip>``: the host's IP. I.e: ``10.0.0.2``. The value ``0.0.0.0`` is an acceptable IP address and will bind to all network interfaces.
```
Comment (alberpilot) [inline source/_templates/installations/elastic/common/elastic-multi-node/configure_elasticsearch.rst]: Please change all `I.e` by `e.g`.

File: source/_templates/installations/elastic/common/generate_elastic_credentials.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
   # /usr/share/elasticsearch/bin/elasticsearch-setup-passwords auto
 
-The ``elastic``'s password must be saved.
+The ``elastic's`` password must be saved.
```
Comment (alberpilot) [inline source/_templates/installations/elastic/common/generate_elastic_credentials.rst]: Not sure about that. The user is `elastic`, so de `'s` need to be out of the format.

File: source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst [hunk 3]
```diff
@@ -83,7 +83,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. [*Master*] This step implies the selection of the Wazuh cluster installation type. In case of configuring a Wazuh single node cluster, the ``Wazuh single-node cluster`` tab must be selected. In case of having two or more nodes, the ``Wazuh multi-node cluster`` tab must be choosen.
+#. [*Master*] This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` and ``Wazuh multi-node cluster`` tab depending on your intallation.
```
Comment (alberpilot) [inline source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst]: Maybe we need to specify what is the difference between both installations. Somebody who doesn't know the difference between Wazuh single-node and multi-node could be grateful if we offer here a little help.

**Comment by dkempny** [comment]
Hello Team,
the above requested changes are done.

All the best,
Daria

File: source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst [hunk 4]
```diff
@@ -83,7 +83,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. [*Master*] This step implies the selection of the Wazuh cluster installation type. In case of configuring a Wazuh single node cluster, the ``Wazuh single-node cluster`` tab must be selected. In case of having two or more nodes, the ``Wazuh multi-node cluster`` tab must be choosen.
+#. [*Master*] This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` tab, for the Wazuh single node cluster, and ``Wazuh multi-node cluster`` tab in case of having two or more nodes.
```
Comment (alberpilot) [inline source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst]: The last part `more nodes` can be more accurate by `two or more Wazuh cluster nodes`, just for differentiation between elasticsearch nodes and Wazuh nodes.

File: source/installation-guide/elasticsearch-cluster/elasticsearch-single-node-cluster.rst [hunk 5]
```diff
@@ -79,7 +79,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. In this step you must select the type of Wazuh cluster installation that you will set in your environment. Select the ``Wazuh single-node cluster`` tab if you want to configure a single Wazuh node cluster. If your Wazuh cluster will have two or more nodes, select the ``Wazuh multi-node cluster`` tab.
+#. This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` tab, for the Wazuh single node cluster, and ``Wazuh multi-node cluster`` tab in case of having two or more nodes.
```
Comment (alberpilot) [inline source/installation-guide/elasticsearch-cluster/elasticsearch-single-node-cluster.rst]: The last part more nodes can be more accurate by two or more Wazuh cluster nodes, just for differentiation between elasticsearch nodes and Wazuh nodes.

File: source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst [hunk 6]
```diff
@@ -83,7 +83,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. [*Master*] This step implies the selection of the Wazuh cluster installation type. In case of configuring a Wazuh single node cluster, the ``Wazuh single-node cluster`` tab must be selected. In case of having two or more nodes, the ``Wazuh multi-node cluster`` tab must be chosen.
+#. [*Master*] This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` tab, for the Wazuh single node cluster, and ``Wazuh multi-node cluster`` tab in case of having two or more Wazuh cluster nodes.
```
Comment (alberpilot) [inline source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst]: I think that we have redundancy here: 

> Choose between **Wazuh single-node cluster** tab, for the **Wazuh single node cluster**. 

Something like: Choose between **Wazuh single-node cluster**, for only one Wazuh server instance installation type.

File: source/installation-guide/elasticsearch-cluster/elasticsearch-single-node-cluster.rst [hunk 7]
```diff
@@ -79,7 +79,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. In this step you must select the type of Wazuh cluster installation that you will set in your environment. Select the ``Wazuh single-node cluster`` tab if you want to configure a single Wazuh node cluster. If your Wazuh cluster will have two or more nodes, select the ``Wazuh multi-node cluster`` tab.
+#. This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` tab, for the Wazuh single node cluster, and ``Wazuh multi-node cluster`` tab in case of having two or more Wazuh cluster nodes.
```
Comment (alberpilot) [inline source/installation-guide/elasticsearch-cluster/elasticsearch-single-node-cluster.rst]: Same

File: source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst (hunk 6 above)
Comment (dkempny) [reply source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst]: I will change.

**Comment by dkempny** [comment]
The section is rephrased now.

**Comment by dkempny** [comment]
The latest changes included.

**Pull request merged**